server/routes/auth.js

```diff
@@ -16,17 +18,46 @@ router.post("/register", async (req, res, next) => {
         next(errorMessage);
       } else {
         let hash = await bcrypt.hash(req.body.password, 10);
-        let newUser = new User({
-          username: req.body.username,
-          password: hash
-        });
-        newUser.save(function(err, user) {
+        SettingsModel.findOne({ email: req.params.email }, (err, settings) => {
```

Comment:
Don't need to find a setting since users and settings are 1:1, so if we didn't find the user, we wouldn't have a corresponding settings object for the user as well.

Reply:
I removed it